src/coreclr/jit/lsra.h

```diff
@@ -1042,7 +1042,7 @@ class LinearScan : public LinearScanInterface
     void buildRefPositionsForNode(GenTree* tree, LsraLocation loc);
 
 #if FEATURE_PARTIAL_SIMD_CALLEE_SAVE
-    void buildUpperVectorSaveRefPositions(GenTree* tree, LsraLocation currentLoc, regMaskFloat fpCalleeKillSet);
```

Comment:
Does this need to be `Mixed`? This is for UpperVectorSave, so presumably it only needs to care about the float/simd ones?

Reply:
yes, it should be `regMaskFloat`, it is just that the caller doesn't filter them out.